websites/A/Averge/presence.ts

```diff
@@ -0,0 +1,49 @@
+const presence1 = new Presence({
+  clientId: "1328861612735598664", // Deine Client ID
+}), browsingTimestamp1 = Math.floor(Date.now() / 1000);
+
+presence1.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    startTimestamp: browsingTimestamp1,
+    largeImageKey: "averge-eu-logo", // Dein Image-Key im PreMiD-Dashboard
+  };
+
+  const path = window.location.pathname.toLowerCase();
+
+  if (window.location.href.startsWith("https://averge.eu")) {
```

Comment:
Use `document.location` instead of `window.location` for consistency with other activities